src/test/java/com/commercetools/sync/customobjects/utils/CustomObjectSyncUtilsTest.java

```diff
@@ -0,0 +1,249 @@
+package com.commercetools.sync.customobjects.utils;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import io.sphere.sdk.customobjects.CustomObject;
+import io.sphere.sdk.customobjects.CustomObjectDraft;
+
+import org.junit.jupiter.api.Test;
+
+import javax.annotation.Nonnull;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static com.commercetools.sync.commons.utils.SyncUtils.batchElements;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class CustomObjectSyncUtilsTest {
+
+    private CustomObject<JsonNode> oldCustomObject;
+    private CustomObjectDraft<JsonNode> newCustomObjectdraft;
```

Comment:
Why do you have those as class vars and not method vars?

Reply:
They are used throughout the whole test class.

Reply:
But you don't share them between the methods, so I think it's better to create a method variable.

Reply:
I refactored the class. Now `key` and `container` are local variables in a private method
https://github.com/commercetools/commercetools-sync-java/pull/561/commits/ac2a28b3d492582f40a9590d2019738d9287301d